deployment/terraform/cdn.tf

```diff
@@ -61,6 +61,28 @@ resource "aws_cloudfront_distribution" "cdn" {
     max_ttl                = 300
   }
 
+  ordered_cache_behavior {
+    path_pattern     = "tile/*"
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originAlb"
+
+    forwarded_values {
+      query_string = false
+      headers      = ["*"]
```

Comment:
Slightly similar comment here, except that I don't think we want to pass any headers through. Any differences in the values of headers across requests will cause objects to be cached separately within CloudFront. We need to evaluate all of the relevant headers and only includes ones (if any) in this list that we want to be cached separately.